commands/pm/backdrop_pm_disable.drush.inc

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Drush project management disable command.
+ */
+
+Use \Drupal\Core\Logger\RfcLogLevel;
```

Comment:
Hm, seems like this would likely cause a problem in a Backdrop repo? Do we use `RfcLogLevel` constants or functions anywhere in here?